Gruntfile.js

```diff
@@ -403,4 +379,13 @@ module.exports = function (grunt) {
     grunt.registerTask("deploy-branch", ["default","gitinfo", "clean:current_branch", "copy:copy_current_branch", "gh-pages:deploy-branch"]);
     /* clean all the files in gh-pages branch */
     grunt.registerTask("gh-pages-clean", ["gh-pages:clean"]);
+
+    // conditional switch for deployment
+    grunt.registerTask("deploy", function (deployment_mode) {
```

Comment:
I think we don't need the `deployment_mode` parameter, checking the `TRAVIS_BRANCH` should be enough